frame/fast-unstake/src/lib.rs

```diff
@@ -46,16 +50,29 @@
 //!
 //! If unsuccessful, meaning that the staker was exposed sometime in the last `BondingDuration` eras
 //! they will end up being slashed for the amount of wasted work they have inflicted on the chian.
+//!
+//! ## Details
+//!
+//! See [`pallet`] module for more information.
+//!
+//! ## Examples
+//!
+//! 1. Fast-unstake with multiple participants in the queue.
+#![doc = docify::embed!("./frame/fast-unstake/src/tests.rs", successful_multi_queue)]
```

Comment:
Currently the example is not compiled. Is this intended @sam0x17 ?

Reply:
Yeah, and I think it will always be the case. The only benefit of the `examples` approach was that we would have gotten a compiling example, but that required leaking `mock` crate to `std`. I personally like this approach better. Sure, the code is marked as `does not compile`, but we actually know that it does. 

It would be good to auto-generate a sentence at the bottom of these, saying "this snippet is taken from <file-name>".

Reply:
> Yeah, and I think it will always be the case. The only benefit of the `examples` approach was that we would have gotten a compiling example, but that required leaking `mock` crate to `std`. I personally like this approach better. Sure, the code is marked as `does not compile`, but we actually know that it does.
> 
> It would be good to auto-generate a sentence at the bottom of these, saying "this snippet is taken from ".

There is an option to have it compile if you like if you check the docs for `docify::embed!`, by default it is disabled.

It should still work @KiChjang  if you CD because the working directory for all crates in a workspace is still the workspace root regardless of where you are running `cargo build` from. If this wasn't the case, a lot of other things would frequently break.

Reply:
I'm having trouble getting `cargo doc` or `cargo b` to run even from the project root. 

It fails asking for a path relative to the `fast-unstake` pallet:

<img width="1039" alt="Screenshot 2023-05-24 at 09 49 50" src="https://github.com/paritytech/substrate/assets/16665596/6d46dd22-d2c4-4b61-bd07-efae5080add1">

but when I update the path accordingly, it now wants it relative to the root:

<img width="851" alt="Screenshot 2023-05-24 at 09 53 28" src="https://github.com/paritytech/substrate/assets/16665596/c1cb881e-ec69-473e-a9a5-c4d5f5997569">

maybe I'm doing something wrong?